core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -200,6 +216,41 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
     case Terminated(ref) =>
       requests -= ref
       requestors -= ref
+
+    case Metadata.ListTopics =>
+      sender() ! Metadata.Topics(Try {
+        consumer.listTopics().asScala.map {
```

Comment:
You really should not block this actor, it is too crucial in the processing

For example: Imagine a client, which checks for topics like every minute, for a big kafka cluster with lots of queues, it might stop the processing for a second or more, which is pretty unexpected, given an asynchronous actor messaging API.

Reply:
https://kafka.apache.org/0100/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#multithreaded